portal-wire-protocol.md

```diff
@@ -68,6 +68,39 @@ The transmission of data that is too large to fit a single packet is done using
 
 > The Portal wire protocol currently implements uTP over the `TALKREQ/TALKRESP` messages.  Future plans are to move to the [sub-protocol data transmission](https://github.com/ethereum/devp2p/issues/229) in order to use a protocol native mechanism for establishing packet streams between clients.
 
+Currently, the standard is to switch to uTP when the payload exceeds 1165 bytes. This may change over time, because it depends on a number of other variables. See an example derivation in rust:
```

Comment:
Great for adding this in the specifications, but I believe the value should be 1177 bytes.

That's the value that gets calculated by Fluffy at least, and IIRC this was at some point in the past also looked at as reference by Trin, but perhaps it was not taken over exactly or got altered over time.

Calculation in Fluffy happens here: https://github.com/status-im/nimbus-eth1/blob/master/fluffy/network/wire/portal_protocol.nim#L123 & https://github.com/status-im/nimbus-eth1/blob/master/fluffy/network/wire/portal_protocol.nim#L404-L406

I don't have an exact test for this, but I do remember verifying this byte per byte on a encoded `talkresp` message.
I will make the `const`s more clear and can add a test for this also to show.

Reply:
I have moved the calculation for talkreq overhead to our discv5 module and added tests there: https://github.com/status-im/nim-eth/commit/aa1e738a97d1995c1ddb821123e4842409218b0c

So I'm pretty certain that these are the correct values (at least with our discv5 impl., but that shouldn't differ really).

Reply:
It looks like this test is for the max size of the Talk Response payload, but the relevant payloads for a Content message are transported in a Talk Request. Does that make a difference?

Reply:
The  content message adds just two additional bytes of overhead, so those need to be added still:

https://github.com/status-im/nimbus-eth1/blob/master/fluffy/network/wire/portal_protocol.nim#L391

You can also see that in our test vectors: https://github.com/ethereum/portal-network-specs/blob/master/portal-wire-test-vectors.md#content-response---content-payload

Reply:
Ok, I'll update the spec to note that we're talking about the size of the encoded message that includes the message ID, and leave out the extra 2 bytes. I don't see a test in trin for this content size max, so I'll take your values here :)

Reply:
Wait, but it looks like the message overhead is different for the different scenarios, like find nodes vs find content, so maybe I'll just ignore the 2 bytes here, and look at the total encoded payload size

Reply:
Yes, it will be different for each of the wire protocol messages. But I guess the most important one here is the one that differentiates between sending content over the protocol response or over uTP